app/main.py

```diff
@@ -2,0 +1,18 @@
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError, VaccineError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    masks_needed = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks_needed += 1
+
+    if masks_needed:
+        return f"Friends should buy {masks_needed} masks"
+
+    return f"Friends can go to {cafe.name}"
```

Comment:
The function `go_to_cafe` is missing a function annotation for its return type. According to the checklist, it's good practice to use annotations. This not only makes the code easier to understand, but also helps with static type checking. Please add the appropriate annotation for the return type of this function.